README.md

```diff
@@ -186,7 +186,11 @@ my-angular-workspace/
 
 ### Component Analysis
 
-- **`report-violations`**: Report deprecated CSS usage in a directory with configurable grouping format
+- **`report-violations`**: Report deprecated CSS usage for a specific DS component in a directory. Supports optional file output with statistics (components, files, lines).
```

Comment:
The tool should be usage-agnostic as much as possible, the previous versions were more neutral and not DS specific